CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ different versioning scheme, following the Haskell community's
 * C++ codegen now can use [`std::scoped_allocator_adaptor`](http://en.cppreference.com/w/cpp/memory/scoped_allocator_adaptor)
   for strings and containers when custom allocator is used and `--scoped-alloc` flag
   is passed to `gbc`.
+* C++ codegen now generates lazily constructible enum maps and accepts user-defined
```

Comment:
Minor: how about "C++ codegen now generated lazily constructed enum name-to-value and value-to-name maps. Additionally, a user-defined map type can now be provided to `GetNameToValueMap` and `GetValueToNameMap`."?

The unordered_map change is an implementation detail, so it can be omitted from the changelog. If you want to talk about it, maybe something like "Enum name-to-value and value-to-name look ups now work in amortized constant time."?

Reply:
Updated.